typings/es2023-shim/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// TODO: Remove this when we upgrade to TypeScript 5
```

Comment:
https://github.com/microsoft/fluentui/issues/31857

Reply:
Added the issue URL to the comment.